codequality/README.md

```diff
@@ -49,14 +49,14 @@ $ nuget
 
 #### Creating the local repo
 
-1. Compile the sources of the aerogear-xamarin-sdk as debug
+1. Compile the sources of the `aerogear-xamarin-sdk` as `debug`
```

Comment:
Maybe we should say "In Visual Studio" or give the example msbuild command?

Reply:
done